chai-immutable.js

```diff
@@ -540,5 +540,97 @@
     assert.sizeOf = function (collection, expected) {
       new Assertion(collection).size(expected);
     };
+
+    /**
+     * ### .property(path, value)
```

Comment:
For alphabetical sorting consistency, would you mind moving this addition [between `keys` and `size`, in the BDD reference section of the code](https://github.com/astorije/chai-immutable/blob/25a7a6d125f0e3217069782127f1961d11aabd41/chai-immutable.js#L282-L286) please?